frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -170,7 +166,7 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
             position={[latitude, longitude]}
             icon={getMarkerIcon(cluster)}
           >
-            <Popup>
+            <Popup autoPan={false}>
```

Comment:
What does this option do?

Reply:
Stops the map from refocussing to an open marker popup